src/components/Dashboard/FilingHistoryList/filings/ContinuationInApplication.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <FilingTemplate
+    class="continuation-in-application"
+    :filing="filing"
+    :index="index"
+  >
+    <template #subtitle>
+      <FiledAndPendingPaid
+        v-if="isFutureEffectivePending"
+        class="item-header-subtitle"
+        :filing="filing"
+        :index="index"
+      />
+
+      <FutureEffectivePaid
+        v-else-if="isFutureEffective"
+        class="item-header-subtitle"
+        :filing="filing"
+        :index="index"
+      />
```

Comment:
See if you can test both of the above cases ^^

Reply:
> See if you can test both of the above cases ^^ and their corresponding bodies below.

I will add the tests tmr before I merge this PR. Thank you!

Reply:
No, I mean manually verify the screen. You'll have to create a future-effective continuation in to see this.

Reply:
> No, I mean manually verify the screen. You'll have to create a future-effective continuation in to see this.



> No, I mean manually verify the screen. You'll have to create a future-effective continuation in to see this.

Got it, thank you. I'm able to test the 2 cases, it shows properly:
future-effective:
<img width="725" alt="1" src="https://github.com/bcgov/business-filings-ui/assets/122495122/78487c5f-5f70-4d60-a65c-743e774c4be7">

future-effective pending:
<img width="759" alt="2" src="https://github.com/bcgov/business-filings-ui/assets/122495122/ec96aef0-b69d-4e4a-8bfe-4f00541dabf9">

Reply:
Looks great. Resolving.

Go ahead and merge.